File: app/src/main/java/com/cabe/app/novel/activity/SplashActivity.java
package com.cabe.app.novel.activity;

import android.content.Intent;
import android.os.Bundle;

import com.cabe.app.novel.R;

public class SplashActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        startActivity(new Intent(this, HomeActivity.class));
        finish();
    }
}

File: app/src/main/java/com/cabe/app/novel/activity/SplashActivity.kt
package com.cabe.app.novel.activity

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import com.cabe.app.novel.R

@SuppressLint("CustomSplashScreen")
class SplashActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        startActivity(Intent(this, HomeActivity::class.java))
        finish()
    }
}